assemble/conf/log4j-monitor.properties

```diff
@@ -15,18 +15,18 @@
 
 # Write out everything at the DEBUG level to the debug log
 log4j.appender.A2=org.apache.log4j.RollingFileAppender
-log4j.appender.A2.File=${org.apache.accumulo.core.dir.log}/${org.apache.accumulo.core.application}_${org.apache.accumulo.core.ip.localhost.hostname}_fromprops.debug.log
+log4j.appender.A2.File=${accumulo.log.dir}/${accumulo.application}_${accumulo.local.hostname}.debug.log
 log4j.appender.A2.MaxFileSize=100MB
-log4j.appender.A2.MaxBackupIndex=10
+log4j.appender.A2.MaxBackupIndex=5
 log4j.appender.A2.Threshold=DEBUG
 log4j.appender.A2.layout=org.apache.log4j.PatternLayout
 log4j.appender.A2.layout.ConversionPattern=%d{ISO8601} [%-8c{2}] %-5p: %X{application} %m%n
 
 # Write out INFO and higher to the regular log
 log4j.appender.A3=org.apache.log4j.RollingFileAppender
-log4j.appender.A3.File=${org.apache.accumulo.core.dir.log}/${org.apache.accumulo.core.application}_${org.apache.accumulo.core.ip.localhost.hostname}_fromprops.log
+log4j.appender.A3.File=${accumulo.log.dir}/${accumulo.application}_${accumulo.local.hostname}.log
```

Comment:
We definitely don't need both the "regular log" and "debug log" configured in the default configuration file. We could shorten, and dramatically simplify the default file by showing only one example, and indicating that appenders could be used with different settings. This makes the file easier to understand for users who wish to dive in and customize.